WcaOnRails/spec/controllers/teams_controller_spec.rb

```diff
@@ -66,6 +66,27 @@
       get :edit, params: { id: Team.banned.id }
       expect(response).to render_template :edit
     end
+
+    it "senior delegates can manage the probation team, despite not being a member of the probation team" do
+      sign_in FactoryBot.create :senior_delegate
+
+      get :edit, params: { id: Team.probation.id }
+      expect(response).to render_template :edit
+    end
+
+    it "leader of WFC can manage the probation team, despite not being a member of the probation team" do
+      sign_in FactoryBot.create :user, :wfc_leader
```

Comment:
This should be: `sign_in FactoryBot.create :user, :wfc_member, team_leader: true`